Build error  (macOS 10.15 Catalina)

I tried to build, but it gives an error at the end:

```
MACOS Installation from source
 ------------------------
 brew install gr-osmosdr
 brew install gnuradio
 brew install cmake
 brew install qt


$ git clone https://github.com/csete/gqrx.git gqrx.git
$ cd gqrx.git
$ mkdir build
$ cd build
$ cmake ..
$ make
```

error:

```
[ 90%] Building CXX object src/CMakeFiles/gqrx.dir/receivers/nbrx.cpp.o
[ 92%] Building CXX object src/CMakeFiles/gqrx.dir/receivers/receiver_base.cpp.o
[ 93%] Building CXX object src/CMakeFiles/gqrx.dir/receivers/wfmrx.cpp.o
[ 94%] Building CXX object src/CMakeFiles/gqrx.dir/portaudio/device_list.cpp.o
[ 96%] Building CXX object src/CMakeFiles/gqrx.dir/portaudio/portaudio_sink.cpp.o
[ 97%] Building CXX object src/CMakeFiles/gqrx.dir/qrc_icons.cpp.o
[ 98%] Building CXX object src/CMakeFiles/gqrx.dir/qrc_textfiles.cpp.o
[100%] Linking CXX executable gqrx
Undefined symbols for architecture x86_64:
  "_volk_32f_log2_32f", referenced from:
      MainWindow::iqFftTimeout() in mainwindow.cpp.o
  "_volk_32f_s32f_multiply_32f", referenced from:
      MainWindow::iqFftTimeout() in mainwindow.cpp.o
  "_volk_32fc_magnitude_squared_32f", referenced from:
      MainWindow::iqFftTimeout() in mainwindow.cpp.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [src/gqrx] Error 1
make[1]: *** [src/CMakeFiles/gqrx.dir/all] Error 2
make: *** [all] Error 2
Apple@iMac-Apple build % 
```

**How to fix it?**

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -71,6 +71,7 @@ target_link_libraries(${PROJECT_NAME}
     ${PULSEAUDIO_LIBRARY}
     ${PULSE-SIMPLE}
     ${PORTAUDIO_LIBRARIES}
+    volk
 )
 
 if(NOT Gnuradio_VERSION VERSION_LESS "3.8")
@@ -80,6 +81,7 @@ if(NOT Gnuradio_VERSION VERSION_LESS "3.8")
         gnuradio::gnuradio-digital
         gnuradio::gnuradio-filter
         gnuradio::gnuradio-audio
+        volk
     )
 endif()
 

